### DISCUSSION TITLE
Weird font rendering

### DISCUSSION THREAD

**Discussion by davidnoronha1** [wezterm/wezterm#643] [Q&A] [answered]
Wezterm's font looks a bit glitchy
Default works fine for some reason but anything set from wezterm.lua looks a bit glitchy
Comparison:
st / wezterm with Sarasa Term K ~ Ubuntu Mono also glitches
![1617604236](https://user-images.githubusercontent.com/38278035/113545092-297e1b80-9607-11eb-8641-ee51cb335100.png)
![1617604234](https://user-images.githubusercontent.com/38278035/113545103-2e42cf80-9607-11eb-9d64-4a6b1a0da28c.png)

This never happened to me with wezterm before so I have no idea what it might be
Installed it from the deb file of the latest release

Great terminal btw
St seems to have a bad habit of crashing xorg whenever if feels like it :sweat_smile: 
I tried messing with lineheight to no avail

**Comment by wez** [comment]
Weird.  I can't reproduce this on macos with that same font.
You mentioned you hadn't seen that before with wezterm?
I wonder if you have `FREETYPE_PROPERTIES` set in the environment when wezterm is started?

**Reply by davidnoronha1** [reply]
no that doesnt seem to be it
Its a bit weird that the default works just fine but not anything else :thinking: 
ill try compiling it if that helps at all but maybe not 

I think I messed about with libxft for st a bit maybe ill revert that back

**Reply by davidnoronha1** [reply]
I had a similar issue when I tried forcing bitmaps to work in kitty that wast fun
iirc line height or some cell size settings helped

**Comment by davidnoronha1** [comment, marked as answer]
Fixed now
